rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java

```diff
@@ -793,48 +790,30 @@ private boolean setDropZoneInfo() throws Exception
         boolean success = false;
 
         HPCCFileSprayClient fsclient = getFileSprayClient();
-
-        DropZoneFilesRequest dropzonefilesreq = new DropZoneFilesRequest();
-
         try
         {
-            DropZoneFilesResponse dropZoneFilesResponse = fsclient.getSoapProxy().dropZoneFiles(dropzonefilesreq);
+        	targetDropzoneNetAddres = fsclient.fetchLocalDropZones()[0].getNetAddress();
+            Utils.println(System.out, "Found dropzone net address: " + targetDropzoneNetAddres, false, verbosemode);
 
-            if (dropZoneFilesResponse.getExceptions() != null)
+        	targetHPCCDropzonePath = fsclient.fetchLocalDropZones()[0].getPath();
```

Comment:
rather than making multiple calls to fsclient.fetchLocalDropZones() (line 795, 798) let's make the call once and store result in local var.